qml-ui/main.qml

```diff
@@ -680,388 +680,443 @@ Kirigami.AbstractApplicationWindow {
-            property QtObject currentSequence: Zynthbox.PlayGridManager.getSequenceModel(zynqtgui.sketchpad.song.scenesModel.selectedSequenceName)
-            onClicked: {
-                // handle live-recording-is-going state here, otherwise you might turn it
-                // on in the sequencer, then head out, and try and turn it off and it just
-                // opens the recording popup, which isn't what you'd be after
-                if (globalRecordButton.currentSequence.activePatternObject && globalRecordButton.currentSequence.activePatternObject.recordLive) {
-                    globalRecordButton.currentSequence.activePatternObject.recordLive = false;
-                    if (Zynthbox.PlayGridManager.metronomeActive) {
-                        Zynthian.CommonUtils.stopMetronomeAndPlayback();
+                        icon.name: "media-record-symbolic"
+                        icon.width: 24
+                        icon.height: 24
+                        icon.color:  globalRecordButton.currentSequence.activePatternObject && globalRecordButton.currentSequence.activePatternObject.recordLive
+                                     ? "#ff5cf436" // A green with the same values as the red audio record colour below
+                                     : zynqtgui.sketchpad.isRecording ? "#fff44336" : Kirigami.Theme.textColor
+                    }
+
+                    QQC2.Button {
+                        id: globalPlaybackButton
+                        width: Kirigami.Units.gridUnit*4
+                        height: parent.height
+                        onClicked: {
+                            if (zynqtgui.sketchpad.isMetronomeRunning) {
+                                zynqtgui.callable_ui_action_simple("SWITCH_STOP");
+                            } else {
+                                zynqtgui.callable_ui_action_simple("SWITCH_PLAY");
+                            }
+                        }
+
+                        icon.name: zynqtgui.sketchpad.isMetronomeRunning ? "media-playback-stop" : "media-playback-start"
+                        icon.width: 24
+                        icon.height: 24
+                        icon.color: Kirigami.Theme.textColor
                     }
-                } else {
-                    applicationWindow().openRecordingPopup();
                 }
             }
 
-            Kirigami.Icon {
-                width: Kirigami.Units.gridUnit
-                height: width
-                anchors.centerIn: parent
-                source: "media-record-symbolic"
-                color: globalRecordButton.currentSequence.activePatternObject && globalRecordButton.currentSequence.activePatternObject.recordLive
-                    ? "#ff5cf436" // A green with the same values as the red audio record colour below
-                    : zynqtgui.sketchpad.isRecording ? "#fff44336" : "white"
-            }
-        }
-        QQC2.Button {
```

Comment:
The playback buttons were using an overlay icon, which duplicates the Button's own icon implementation, so I removed the duplication and instead started using the Button icon.